packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -911,6 +959,14 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
       while (nextUnitOfWork !== null && !shouldYield()) {
         nextUnitOfWork = performUnitOfWork(nextUnitOfWork);
       }
```

Comment:
Running the base timer for everything in the tree regardless of if this fiber is part of a profiled tree or not makes this a cost paid by everyone, whether you use the feature or not. This is probably fine for now since it's off by default.

If we want to turn it on my default, then we should probably move the `startBaseRenderTimer` here. That way we can start a work loop that doesn't have a `startBaseRenderTimer` call in it, unless the first fiber is already in a profiled tree. Then when we hit a Profiler fiber, we can start another work loop that has the `startBaseRenderTimer` call in it.

We can do that later though. No need to do it in this PR.